Cartfile

```diff
@@ -5,3 +5,4 @@ github "kif-framework/KIF"
 github "st3fan/SWXMLHash"
 github "rs/SDWebImage"
 github "norio-nomura/Base32"
+github "DaveWoodCom/XCGLogger" ~> 1.8
```

Comment:
Isn't that what `Cartfile.resolved` is for? When you run `carthage checkout`, the versions are still pinned to `Cartfile.resolved` and won't change until you run `carthage update`.